test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -2814,6 +2815,7 @@ Expect.describe('PrimaryAuth', function () {
             return Expect.waitForWindowListener('message', test);
           })
           .then(function (test) {
+            jasmine.clock().mockDate(new Date(AUTH_TIME + 10000));
```

Comment:
Dont we have to do jasmine.clock install and uninstall to make sure we clean up ?Can you also add a comment please

Reply:
yes, you are right. Added cleanup code and comments.